pkg/minikube/machine/delete.go

```diff
@@ -40,23 +40,30 @@ func deleteOrphanedKIC(ociBin string, name string) {
 		return
 	}
 
+	defer func() { // networks should be deleted after containers
```

Comment:
why the defer? We have no idea at this point if we were successfully able to delete the container.

Reply:
beause in the code bellow somwhere we return if container doesnt exist. so I use defer.
and also you can not delete network if container is attached to it

Reply:
and this is a clean up code, so we just try and if it doesnt get deleted, we at least have tried.

Reply:
I guess that's the part I don't understand: if this fails unless the container is deleted (the final step), just simplify this and move it to occur explicitly afterwards. 

If this defer is really fixing a specific behavior, please document it in the comment and/or propose it in another PR.